R/read_nmext.R

```diff
@@ -28,19 +39,24 @@ read_nmext <- function(run=NA_real_, project = getwd(), file=paste0(run,".ext"),
   
   use_dt <- requireNamespace("data.table",quietly=TRUE) & read_fun=="data.table"
   
-  ##Check for multiple tables
-  ext_tmp <- readr::read_lines(extfile,n_max=-1)
+  ## Check for multiple tables/problems
+  
+  ext_tmp <- readr::read_lines(extfile,n_max=-1) ## May be substituted for another read function, so package does not depend on readr
```

Comment:
Use `fread` here please;  That way you don't add another dependency.

Reply:
I couldn't figure out how to use `fread`, however no i am using `readLines`, which is a base R function